spacewalk/certs-tools/mgr_ssl_cert_setup.py

```diff
@@ -300,24 +300,16 @@ def getCertWithText(cert):
 
 
 def getPrivateKey(key):
-    if "-----BEGIN RSA PRIVATE KEY-----" in key:
-        privatekeytype = "rsa"
```

Comment:
`privatekeytype` is a global variable and used also at other places.
Please remove it also there.